SHA256SUMS

```diff
@@ -1,7 +1,11 @@
 700df0d3013b703a806d2ae7f1bfb8e59814e3d06ae78be0c66368a50059f33d  models/7B/consolidated.00.pth
+0cc0b0a3dc8cd29f005946f8364ac2bbce797e792a40c0fb4114615e4f825976  models/7B/ggml-model-f16.bin
+5dec1979849d73e361a8bcc10bc8f53237cbbe435a572882dc87629e011e24b3  models/7B/ggml-model-q4_0.bin
```

Comment:
Could you please remove quantized models, since everyone would have their unique quantized models.

Reply:
OK, I have generated new quantized model and checksum mathes with yours.

Reply:
I am sorry, is this checksum is for q4_0 that have no minor version yet?